cmd/context.go

```diff
@@ -109,6 +109,28 @@ var clearContextCmd = &cobra.Command{
 	},
 }
 
+var loginContextCmd = &cobra.Command{
+	Use:     "login <context-name> <username> <password>",
+	Args:    cobra.RangeArgs(1, 3),
+	Short:   "Login and set cookie for the named context",
+	Long:    "Login and set cookie for the named context",
+	Example: "corectl context login",
```

Comment:
Should probably add more examples since the arguments are not mandatory

Reply:
Only contextName is an argument now. user and password is localflags